Formula/ble-sh.rb

```diff
@@ -0,0 +1,37 @@
+class BleSh < Formula
+  desc "Bash Line Editor: syntax highlighting, auto suggestions, vim modes, etc."
+  homepage "https://github.com/akinomyoga/ble.sh"
+  url "https://github.com/akinomyoga/ble.sh/archive/v0.3.2.tar.gz"
+  sha256 "c0b9a1521e375bcc38252cb0863404a246fe54c1619c58657253698220e78c68"
+  head "https://github.com/akinomyoga/ble.sh.git"
+
+  keg_only "this formula puts files in lib that shouldn't be linked"
+
+  depends_on "gawk" => :build
+  depends_on "make" => :build
+
+  uses_from_macos "bash"
+
+  def install
+    system Formula["make"].bin/"gmake"
+    system Formula["make"].bin/"gmake", "INSDIR=#{prefix}", "install"
```

Comment:
> audit fails with only one error, and that one is unavoidable: there are files in lib that shouldn't be there. Unfortunately, for this particular software, they should be there, and if they're moved then the top-level script can't find them anymore.

The directory `lib` in tarball does not corresponds to the directory something like `/usr/local/lib` but to `/usr/local/share/blesh/lib`. I'm not familiar with homebrew philosophy, but I guess this should be something like the following:

```suggestion
    system Formula["make"].bin/"gmake", "INSDIR=#{prefix}/share/blesh", "install"
```